framework/test/unit/modules/nft/endpoint.spec.ts

```diff
@@ -360,80 +359,70 @@ describe('NFTEndpoint', () => {
 		});
 	});
 
-	describe('getCollectionIDs', () => {
-		it('should fail if provided chainID has invalid length', async () => {
-			const minLengthContext = createTransientModuleEndpointContext({
-				stateStore,
-				params: {
-					chainID: utils.getRandomBytes(LENGTH_CHAIN_ID - 1).toString('hex'),
-				},
+	describe('getSupportedCollectionIDs', () => {
+		it('should return a supportedCollectionIDs array as [*] when ALL_SUPPORTED_NFTS_KEY exists in SupportedNFTsStore', async () => {
+			await supportedNFTsStore.save(methodContext, ALL_SUPPORTED_NFTS_KEY, {
+				supportedCollectionIDArray: [],
 			});
 
-			const maxLengthContext = createTransientModuleEndpointContext({
+			const context = createTransientModuleEndpointContext({
 				stateStore,
-				params: {
-					chainID: utils.getRandomBytes(LENGTH_CHAIN_ID + 1).toString('hex'),
-				},
 			});
 
-			await expect(endpoint.getCollectionIDs(minLengthContext)).rejects.toThrow(
-				`'.chainID' must NOT have fewer than 8 characters`,
-			);
-
-			await expect(endpoint.getCollectionIDs(maxLengthContext)).rejects.toThrow(
-				`'.chainID' must NOT have more than 8 characters`,
-			);
+			await expect(endpoint.getSupportedCollectionIDs(context)).resolves.toEqual({
+				supportedCollectionIDs: ['*'],
+			});
 		});
 
-		it('should return empty list if provided chainID does not exist', async () => {
+		it('should return a supportedCollectionIDs array with chainID + 8 [*]s when supportedCollectionIDArray is empty', async () => {
```

Comment:
It should also return ownChainID + 8 [*]s in addition to the original expectation

Reply:
This should be an additional another test or the current one is not sufficient?

Reply:
what do you mean ? The expectation of current test is not correct.